Adyen/Core/Payment Methods/PaymentMethod.swift

```diff
@@ -52,4 +52,14 @@ public protocol StoredPaymentMethod: PaymentMethod {
     /// The supported types of shopper interaction.
     var supportedShopperInteractions: [ShopperInteraction] { get }
     
+    /// Indicates whether the shopper is present.
+    var isShopperPresent: Bool { get }
```

Comment:
this has to be in the protocol. if i only add it to the protocol extension - we get a segmentation fault from swift compiler 🤷